parkings/models/parking.py

```diff
@@ -29,17 +32,29 @@ class Parking(TimestampedModelMixin, UUIDPrimaryKeyMixin):
     special_code = models.CharField(
         max_length=10, verbose_name=_("special parking code"), blank=True,
     )
-    time_end = models.DateTimeField(
-        verbose_name=_("parking end time"), db_index=True,
-    )
     time_start = models.DateTimeField(
         verbose_name=_("parking start time"), db_index=True,
     )
+    time_end = models.DateTimeField(
+        verbose_name=_("parking end time"), db_index=True,
+    )
     zone = models.IntegerField(verbose_name=_("zone number"), validators=[
         MinValueValidator(1), MaxValueValidator(3),
     ])
 
+    class Meta:
+        verbose_name = _("parking")
+        verbose_name_plural = _("parkings")
+
     def __str__(self):
         start = localtime(self.time_start).replace(tzinfo=None)
         end = localtime(self.time_end).time().replace(tzinfo=None)
         return "%s -> %s (%s)" % (start, end, self.registration_number)
+
+    def is_valid(self):
+        return self.time_start <= now() <= self.time_end
+
+    def get_state(self):
+        if self.time_start <= now() <= self.time_end:
```

Comment:
Actually `is_valid()` is a leftover from a previous implementation and should not be there at all :D 

About properties, personally I prefer methods in these where the value needs to be calculated from other values to distinguish from normal attributes.